drivers/atwinc15x0/atwinc15x0_netdev.c

```diff
@@ -337,6 +356,19 @@ static int _atwinc15x0_recv(netdev_t *netdev, void *buf, size_t len, void *info)
     return rx_size;
 }
 
+static netopt_enable_t _get_link_state(atwinc15x0_t *dev)
+{
+    if (dev->state != NETOPT_STATE_IDLE) {
+        return NETOPT_DISABLE;
+    }
+
+    if (dev->connected) {
+        return NETOPT_ENABLE;
+    }
+
+    return NETOPT_DISABLE;
```

Comment:
The only case where it returns `NETOPT_ENABLE` is when `dev->connected` is true. So it could be simplified if you want:
```suggestion
    return (dev->connected) ? NETOPT_ENABLE : NETOPT_DISABLE;
```

Reply:
It should also return DISABLED when the radio is in sleep mode.
Setting `connect` to false in this case would mess with re-connect after wake-up (I tried that first)

Reply:
Ah ok.